MyBlog.Services/Areas/Users/IUserService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MyBlog.Infrastructure.Entities;
+
+namespace MyBlog.Service.Areas.Users;
+
+public interface IUserService
+{
+    Task<List<User>> GetAllAsync();
+
+    Task<User> GetByIdAsync(int id);
+
+    Task CreateAsync(User userInput);
+    
+    Task UpdateByIdAsync(int id, User userInput);
```

Comment:
in the case with Create and Update, as user, i wanna see changes in the response (i change name and mobile phone for example, and wanna see, that my changes is aply). So, return Created and Updated entity to the user in the response, or, for example, return created and updated userId, and call GetByIdAsync method in your controller, so you can retrieve finally entity and show changes.